packages/grafana-ui/src/components/Table/RowsList.tsx

```diff
@@ -218,6 +237,18 @@ export const RowsList = (props: RowsListProps) => {
         return colors;
       };
     }
+
+    // TODO: Also add option for colored background cell,
```

Comment:
do we have a follow-up github issue capturing this?

Reply:
Actually added this 😄, probably want to also hit colored text too but I can make an issue for that